javascript-light-bulb/main.js

```diff
@@ -0,0 +1,16 @@
+var $circle = document.querySelector('.circle');
+var $background = document.querySelector('body');
+var counter = null;
+
+$circle.addEventListener('click', function (event) {
+  counter++;
```

Comment:
Rather than using a counter variable then checking if the value is even or odd, I would suggest using a boolean to keep track of if the light is currently on or off, and re-assign the value to the opposite boolean every time the user clicks the light switch.